packages/backend/src/api/resources/JSON.ts

```diff
@@ -127,6 +127,8 @@ export interface OrganizationJSON extends ClerkResourceJSON {
   name: string;
   slug: string | null;
   logo_url: string | null;
+  membersCount: number;
+  pendingInvitationsCount: number;
```

Comment:
:wrench: The JSON response will return `pending_invitations_count` (snake cased) in some cases. Although this is never returned from BAPI as far as I know.